config/config.go

```diff
@@ -525,8 +525,7 @@ func (c *Config) Finalize() {
 
 	if c.LogLevel == nil {
 		c.LogLevel = stringFromEnv([]string{
-			"CT_LOG",
-			"CONSUL_TEMPLATE_LOG",
+			"CONSUL_TEMPLATE_LOG_LEVEL",
```

Comment:
How about we just make it all 3. I really don't like breaking backwards compatibility unless absolutely necessary, but I think the new one is better. Thanks.

Reply:
Yes agree with you. I will do it this week.

Reply:
@eikenb As expected, just adding the new env variable